src/java/fr/paris/lutece/plugins/forms/service/FormResponseService.java

```diff
@@ -56,4 +78,35 @@
         return dataList;
     }
     
+    /**
+     * Saves the form response
+     * 
+     * @param formResponse
+     *            the form response to save
+     */
+    public void saveFormResponse( FormResponse formResponse )
+    {
+    	FormResponseHome.update( formResponse );
+    }
+    
+    @Transactional( FormsConstants.BEAN_TRANSACTION_MANAGER )
```

Comment:
Does not this only works if the object is managed by Spring ?
IMHO, making this class a singleton not managed by Spring is a step in the wrong direction.